classes/PodsAPI.php

```diff
@@ -1203,6 +1203,18 @@ public function get_wp_object_fields( $object = 'post_type', $pod = null, $refre
 					'type'  => 'slug',
 					'alias' => array( 'nicename', 'slug', 'permalink' )
 				),
+				'first_name'  => array(
+					'name'  => 'first_name',
+					'label' => 'First Name',
+					'type'  => 'text',
+					'alias' => array( 'firstname' )
+				),
+				'last_name'  => array(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 7 space(s) between "'last_name'" and double arrow, but found 2.

<sub>hash: 963224a94941353a39a06221f39ca578</sub>